Rubberduck.Parsing/Symbols/IdentifierReference.cs

```diff
@@ -82,15 +84,7 @@ public bool HasExplicitCallStatement()
         }
 
         private Lazy<bool> _hasTypeHint;
```

Comment:
Given that this is only initialized in the ctor, it can be made `readonly` or converted into an expression-bodied statement.